openassessment/xblock/static/js/src/lms/oa_staff_area.js

```diff
@@ -30,20 +31,22 @@
          * Load the staff area.
          */
         load: function() {
-            var view = this;
-
-            // If we're course staff, the base template should contain a section
-            // for us to render the staff area into.  If that doesn't exist,
-            // then we're not staff, so we don't need to send the AJAX request.
-            if ($('.openassessment__staff-area', view.element).length > 0) {
-                this.server.render('staff_area').done(function(html) {
-                    // Load the HTML and install event handlers
-                    $('.openassessment__staff-area', view.element).replaceWith(html);
-                    view.server.renderLatex($('.openassessment__staff-area', view.element));
-                    view.installHandlers();
-                }).fail(function() {
-                    view.baseView.showLoadError('staff_area');
-                });
+            if (this.enabled) {
```

Comment:
`?w=1` to see this change more clearly, I just wrapped the whole block in an if block.